cinema/serializers.py

```diff
@@ -33,7 +33,22 @@ class Meta:
 class MovieSerializer(serializers.ModelSerializer):
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
```

Comment:
You don't need to include image here - this serializer is used for creating new movie without image as per task requirements. You need to include image only on list and detail actions